content/docs/csidriver/installation/helm/isilon.md

```diff
@@ -296,6 +296,52 @@ Deleting a storage class has no impact on a running Pod with mounted PVCs. You c
 
 >Note: If you continue to use the old storage classes, you may not be able to take advantage of any new storage class parameter supported by the driver.
 
+**Steps to create secondary storage class:**
+
+There are samples storage class yaml files available under `samples/storageclass`.  These can be copied and modified as needed. 
+
+1. Copy the `storageclass.yaml` to `storageclass_2.yaml` ( This is just an example, you can rename to file you require. )
```

Comment:
Please use some relevant names for files.

Reply:
fixed